### ISSUE TITLE
Artifact deployed to jcenter was build with Java8

### ISSUE THREAD

**Issue by jraanamo** [cucumber/cucumber-jvm-deps#5]
Was trying to use Cucumber on Android but running on Java7, dexing will crash on:

```
10:25:00.860 [INFO] [org.gradle.api.Project] processing cucumber/deps/com/thoughtworks/xstream/converters/reflection/LambdaConverter.class...

10:25:00.862 [ERROR] [org.gradle.api.Project] Dex: Error converting bytecode to dex:
Cause: java.lang.RuntimeException: Exception parsing classes
    UNEXPECTED TOP-LEVEL EXCEPTION:
    java.lang.RuntimeException: Exception parsing classes
        at com.android.dx.command.dexer.Main.processClass(Main.java:752)
        at com.android.dx.command.dexer.Main.processFileBytes(Main.java:718)
        at com.android.dx.command.dexer.Main.access$1200(Main.java:85)
```

The class seems to have version 52. Tried rebuilding this repo with Java7 but ended up with:

```
[ERROR] MavenReportException: Error while generating Javadoc: 
Exit code: 1 - javadoc: error - invalid flag: -Xdoclint:none

Command line was: /Library/Java/JavaVirtualMachines/_jdk1.7.0_45.jdk/Contents/Home/jre/../bin/javadoc @options @packages

Refer to the generated Javadoc files in '/Users/jsr/scratch/cucumber-jvm-deps/target/apidocs' dir.

org.apache.maven.reporting.MavenReportException: 
Exit code: 1 - javadoc: error - invalid flag: -Xdoclint:none

Command line was: /Library/Java/JavaVirtualMachines/_jdk1.7.0_45.jdk/Contents/Home/jre/../bin/javadoc @options @packages
```

**Comment by brasmusson** [comment]
This should be fixed by #4.

**Comment by jraanamo** [comment]
Thanks. Good to know.
